src/main/java/com/spotify/reaper/storage/cassandra/Migration003.java

```diff
@@ -0,0 +1,93 @@
+
+            PreparedStatement insertRprSchedIdx = session.prepare(
+                    "INSERT INTO repair_schedule_by_cluster_and_keyspace(cluster_name, keyspace_name, repair_schedule_id) VALUES(?, ?, ?)");
+
+            Map<Long,UUID> repairUnitIds = new HashMap<>();
+            Map<Long,String> repairUnitClusters = new HashMap<>();
+            Map<Long,String> repairUnitKeyspaces = new HashMap<>();
+
+            for(Row row : session.execute(QueryBuilder.select().from("repair_unit"))) {
+                UUID uuid = UUIDs.timeBased();
+                repairUnitIds.put(row.getLong("id"), uuid);
+                repairUnitClusters.put(row.getLong("id"), row.getString("cluster_name"));
+                repairUnitKeyspaces.put(row.getLong("id"), row.getString("keyspace_name"));
+
+                session.execute(
+                        insertRprUnit.bind(
+                                uuid,
+                                row.getString("cluster_name"),
+                                row.getString("keyspace_name"),
+                                row.getSet("column_families", String.class),
+                                row.getBool("incremental_repair")));
+            }
+            session.executeAsync("DROP TABLE repair_unit");
+
+            for(Row row : session.execute(QueryBuilder.select().from("repair_schedule"))) {
+                UUID uuid = UUIDs.timeBased();
+                long repairUnitId = row.getLong("repair_unit_id");
+                
+                session.execute(
+                        insertRprSched.bind(
+                            uuid,
+                            repairUnitIds.get(repairUnitId),
+                            row.getString("state"),
+                            row.getInt("days_between"),
+                            row.getTimestamp("next_activation"),
+                            Collections.emptySet(),
+                            row.getInt("segment_count"),
+                            row.getString("repair_parallelism"),
+                            row.getDouble("intensity"),
+                            row.getTimestamp("creation_time"),
+                            row.getString("owner"),
+                            row.getTimestamp("pause_time")));
+
+
+                session.executeAsync(insertRprSchedIdx
+                        .bind(repairUnitClusters.get(repairUnitId), repairUnitKeyspaces.get(repairUnitId), uuid));
+                
+                session.executeAsync(insertRprSchedIdx.bind(repairUnitClusters.get(repairUnitId), " ", uuid));
+                session.executeAsync(insertRprSchedIdx.bind(" ", repairUnitKeyspaces.get(repairUnitId), uuid));
+            }
+
+            session.executeAsync("DROP TABLE repair_unit");
+            session.executeAsync("DROP TABLE repair_schedule");
+
+            LOG.warn("Migration of repair_unit and repair_schedule tables completed.");
+        }
+    }
+
+    private void Migration003(){}
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "Migration003" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this method to prevent any misunderstanding or make it a constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1223)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)